reference/7.1/Microsoft.PowerShell.Utility/Invoke-RestMethod.md

```diff
@@ -982,15 +982,17 @@ using flag notation or combining multiple `WebSslProtocol` options with `-bor`,
 multiple protocols is not supported on all platforms.
 
 > [!NOTE]
-> On non-Windows platforms it may not be possible to supply `'Tls, Tls12'` as an option.
+> On non-Windows platforms it may not be possible to supply `'Tls, Tls12'` as an option. Support for
+> `'Tls13'` is not available on all Operating Systems and will need to be verified on a per
+> Operating System basis.
```

Comment:
Don't think 'operating system' needs to be cased here or the line above.

Reply:
fixed